tekton/serviceaccount.yaml

```diff
@@ -0,0 +1,8 @@
+# The ServiceAccount release-right-meow will be already present on the
+# dogfooding cluster
+
+#apiVersion: v1
+#kind: ServiceAccount
+#metadata:
+#  name: release-right-meow
+---
```

Comment:
If i understand correctly you commented the entire part because its handled on dogfooding cluster but may be this will be used in someother cluster is that correct ?

Reply:
right, this file should go away :stuck_out_tongue:

Reply:
@savitaashture yes :) that was the intent.

@vdemeester I shall remove it. :+1:

Reply:
done